components/logs-table/data-table.tsx

```
@@ -70,24 +74,41 @@ export function DataTable<TData, TValue>({
       value: allStatuses
     }
   ])
+  const effectiveColumnFilters = columnFilters ?? internalColumnFilters
+
   const [columnVisibility, setColumnVisibility] = useState<VisibilityState>({})
   const [globalFilter, setGlobalFilter] = useState("")
 
+  const handleColumnFiltersChange = (updaterOrValue: any) => {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)